provisioning/roles/postgis/vars/debian-8.yml

```diff
@@ -0,0 +1 @@
+postgis_package_name: "postgresql-9.4-postgis-2.1"
```

Comment:
i had to rename this to `Debian-8.yml` for the role to work.

Reply:
> My filesystem lacks case-awareness indeed, sorry about that!

nice one, with the `i had to...`